hack/cherry_pick_pull.sh

```diff
@@ -183,6 +183,15 @@ for pull in "${PULLS[@]}"; do
 done
 gitamcleanup=false
 
+# Re-generate docs (if needed)
+echo
+echo "Regenerating docs..."
+if ! hack/generate-docs.sh; then
+  echo
+  echo "hack/generate-docs.sh FAILED to complete."
+  exit 1
```

Comment:
this also breaks cherry-picks to 1.6 and leaves them with lots of uncommitted generated docs in the working dir